### ISSUE TITLE
test_vfio_user is flaky

### ISSUE THREAD

**Issue by rbradford** [cloud-hypervisor/cloud-hypervisor#5771]
The tests fails before even starting the VM:

```
[2023-09-13T23:35:35.112Z] ---- common_parallel::test_vfio_user stdout ----
[2023-09-13T23:35:35.112Z] thread 'common_parallel::test_vfio_user' panicked at 'assertion failed: exec_host_command_status(\"/usr/local/bin/spdk-nvme/rpc.py nvmf_create_transport -t VFIOUSER\").success()', tests/integration.rs:6619:9
[2023-09-13T23:35:35.112Z] stack backtrace:
[2023-09-13T23:35:35.112Z]    0: rust_begin_unwind
[2023-09-13T23:35:35.112Z]              at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/std/src/panicking.rs:575:5
[2023-09-13T23:35:35.112Z]    1: core::panicking::panic_fmt
[2023-09-13T23:35:35.112Z]              at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/panicking.rs:64:14
[2023-09-13T23:35:35.112Z]    2: core::panicking::panic
[2023-09-13T23:35:35.112Z]              at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/panicking.rs:111:5
[2023-09-13T23:35:35.112Z]    3: integration::common_parallel::setup_spdk_nvme
[2023-09-13T23:35:35.112Z]              at ./tests/integration.rs:6619:9
[2023-09-13T23:35:35.112Z]    4: integration::common_parallel::test_vfio_user
[2023-09-13T23:35:35.112Z]              at ./tests/integration.rs:6654:9
[2023-09-13T23:35:35.112Z]    5: integration::common_parallel::test_vfio_user::{{closure}}
[2023-09-13T23:35:35.112Z]              at ./tests/integration.rs:6648:25
[2023-09-13T23:35:35.112Z]    6: core::ops::function::FnOnce::call_once
[2023-09-13T23:35:35.112Z]              at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/ops/function.rs:507:5
[2023-09-13T23:35:35.112Z]    7: core::ops::function::FnOnce::call_once
[2023-09-13T23:35:35.112Z]              at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/ops/function.rs:507:5
[2023-09-13T23:35:35.112Z] note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
[2023-09-13T23:35:35.112Z] 
[2023-09-13T23:35:35.112Z] 
[2023-09-13T23:35:35.112Z] failures:
[2023-09-13T23:35:35.112Z]     common_parallel::test_vfio_user
```

As a start we should print out the output from running the command if it fails (this could be built into `exec_host_command_status`.

**Comment by rmstoi** [comment]
@rbradford  I get error trying to run test_vfio_user
```
ERROR: This system does not support "RDSEED".
Please check that RTE_MACHINE is set correctly.
EAL: FATAL: unsupported cpu type.
EAL: unsupported cpu type.
```
What could be wrong?

**Comment by likebreath** [comment]
This should be addressed by #6032.
